tests/helper.py

```diff
@@ -2,6 +2,18 @@
 import itertools
 import string
 import collections
+import logging
+import json
+
+# Backward Compatibility
+try:
+    unicode()
+except NameError:
```

Comment:
You could probably combine both try-catch blocks, since if one name error occurs, then most likely the other does too.